README.md

```diff
@@ -32,6 +37,7 @@ nextflow run seqeralabs/nf-aggregate \
     --input run_ids.csv \
     --outdir ./results \
     -profile docker \
+    -r main
```

Comment:
Don't think we need to explicitly set this but may be we do need to change this:
```
manifest.defaultBranch
```

I'll sort it in a follow-up PR.

Reply:
Just did in response to:

```
Project `seqeralabs/nf-aggregate` is currently stickied on revision: main -- you need to explicitly specify a revision with the option `-r` in order to use it
```